### DISCUSSION TITLE
如何获取longbench和leval数据集

### DISCUSSION THREAD

**Discussion by kkkjyu** [open-compass/opencompass#1091] [Community Task]
如题，下载了core版本的数据集，没有找到对应的数据

**Comment by Mor-Li** [comment]
LongBench的数据集应该是Huggingface上面[在线](https://huggingface.co/datasets/yanbingzheng/LongBench)的数据集，使用OpenCompass评测应该会自动从Huggingface上下载并缓存到本地。不包含在OpenCompass提供的打包数据集中。
